src/components/SidebarOptions.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./SidebarOptions.css";
+
+type SidebarOptionsProps = {
+  Icon: React.ElementType;
+  text: string;
+};
+
+const SidebarOptions = ({ Icon, text }: SidebarOptionsProps) => {
```

Comment:
For React components, the recommended way of typing is 

```tsx
const SidebarOptions = ({ Icon, text }): React.FC<SidebarOptionsProps> => {
```

Reply:
Resolved